superset/db_engine_specs/trino.py

```diff
@@ -178,7 +191,7 @@ def prepare_cancel_query(cls, query: Query, session: Session) -> None:
             session.commit()
 
     @classmethod
-    def cancel_query(cls, cursor: Any, query: Query, cancel_query_id: str) -> bool:
+    def cancel_query(cls, cursor: Cursor, query: Query, cancel_query_id: str) -> bool:
```

Comment:
`Cursor` here is trino specific Cursor. `BaseEngineSpec` define cursor as `Any` because each database driver has different `Cursor` class.
If's safe to be used here, see other methods.